modules/check_in/spec/request/v2/appointments_request_spec.rb

```diff
@@ -207,10 +215,12 @@
         end
 
         it 'returns error' do
-          VCR.use_cassette 'check_in/facilities/get_facilities_200' do
-            VCR.use_cassette 'check_in/appointments/get_appointments_500' do
-              VCR.use_cassette 'check_in/map/security_token_service_200' do
-                get "/check_in/v2/sessions/#{id}/appointments", params: { start: start_date, end: end_date }
+          VCR.use_cassette 'check_in/clinics/get_clinics_200' do
+            VCR.use_cassette 'check_in/facilities/get_facilities_200' do
```

Comment:
do we even need these 2 cassettes here?

Reply:
Yeah, we don't need it. I just removed it in [3b0ce1a](https://github.com/department-of-veterans-affairs/vets-api/pull/17642/commits/3b0ce1aae8965571b1f8677904383f8199a65367)